shoop/admin/modules/orders/static_src/create/reducers/lines.js

```diff
@@ -40,20 +40,26 @@ function setLineProperties(linesState, lineId, props) {
 
 function getDiscountsAndTotal(quantity, baseUnitPrice, unitPrice, updateUnitPrice=false) {
     const updates = {};
+
     if (updateUnitPrice) {
-        updates.unitPrice = unitPrice;
+        updates.unitPrice = "" + unitPrice;
```

Comment:
Er... is it really a good idea to store everything as strings in the actual data? :open_mouth: